tools/toolsrc/fill_with_zero/main.c

```diff
@@ -15,8 +15,9 @@ int main(int argc, char** argv){
 	unsigned int size = atoi(argv[2]);
 	printf("%d bytes\n", size);
 	fseek(pf, 0, 2);
-	while(ftell(pf) < size)
+	unsigned int fsize = ftell(pf);
+	while(fsize < size)
```

Comment:
It will be an endless loop. ftell returns current position at the moment, not the pointer to the current position.